modules/misc/fontconfig.nix

```
@@ -15,6 +15,41 @@ let
 
   profileDirectory = config.home.profileDirectory;
 
+  fontconfigFileType = lib.types.submodule (
+    { name, ... }:
+    {
+      options = {
+        enable = lib.mkEnableOption "Whether this font config file should be generated.";
+        text = lib.mkOption {
+          type = lib.types.nullOr lib.types.lines;
+          default = null;
+          description = "Verbatim contents of the config file. If this option is null then the 'source' option must be set.";
+        };
+        source = lib.mkOption {
+          type = lib.types.nullOr lib.types.path;
+          default = null;
+          description = "Config file to source. Alternatively, use the 'text' option instead.";
+        };
+        label = lib.mkOption {
+          type = lib.types.str;
+          default = "name";
```

```suggestion
          default = name;
          defaultText = "<name>";
```
This is clearer and would allow you to omit the `config` section below.

---

fixed in f4d182c786c7e39b77763df4c66a811332f6a76b